dpctl-capi/cmake/modules/FindIntelSycl.cmake

```diff
@@ -49,38 +48,32 @@ elseif(DEFINED ENV{ONEAPI_ROOT})
     else()
         message(FATAL_ERROR "Unsupported system.")
     endif()
-    set(USING_ONEAPI_DPCPP True)
 else()
     message(FATAL_ERROR,
         "Could not locate a DPC++ installation. Either pass the path to a "
-        "custom location using CUSTOM_IntelSycl_INSTALL_DIR or set the "
+        "custom location using DPCTL_DPCPP_HOME_DIR or set the "
         " ONEAPI_ROOT environment variable."
     )
     return()
 endif()
 
 # We will extract the version information from the compiler
-if(USING_ONEAPI_DPCPP)
-    set(dpcpp_cmd "${IntelSycl_ROOT}/bin/dpcpp")
-    set(dpcpp_arg "--version")
-else()
-    set(dpcpp_cmd "${IntelSycl_ROOT}/bin/clang++")
-    set(dpcpp_arg "--version")
-endif()
+set(clangpp_cmd "${IntelSycl_ROOT}/bin/clang++")
```

Comment:
`s\clangpp\clangcpp`

Reply:
clangpp stood for clang++, clangcpp does not :)

Reply:
Aha! then how about `clangxx`

Reply:
Renamed to `clangxx`.